demisto_sdk/commands/init/initiator.py

```diff
@@ -197,21 +225,28 @@ def __init__(
         category: str = "",
         script: bool = False,
         pack: bool = False,
+        xsiam: bool = False,
         author_image: str = "",
         demisto_mock: bool = False,
         common_server: bool = False,
+        is_modeling_rules: bool = False,
+        is_parsing_rules: bool = False,
+        from_version: str = "",
         **kwargs,
     ):
         self.output = output if output else ""
         self.id = id
-
+        self.xsiam = xsiam
         self.is_integration = integration
         self.is_script = script
         self.is_pack = pack
         self.author_image = author_image
         self.demisto_mock = demisto_mock
         self.common_server = common_server
         self.category = category
+        self.is_modeling_rules = is_modeling_rules
+        self.is_parsing_rules = is_parsing_rules
```

Comment:
Why do we need this here?
Why xsiam is not enough? if xsiam then create everything. You don't need to specify modeling and parsing rules specifically (going through all the code)